src/TTImp/Elab/RunElab.idr

```diff
@@ -81,14 +81,17 @@ elabScript fc nest env script@(NDCon nfc nm t ar args) exp
         = do empty <- clearDefs defs
              evalClosure empty val
     elabCon defs "Bind" [_,_,act,k]
-        = do act' <- elabScript fc nest env
+        -- act : Elab A
+        -- k : A -> Elab B
+        -- 1) Run elabScript on act stripping off Elab
+        -- 2) apply k to the result of (1)
+        -- 3) Run elabScript on the result stripping off Elab
+        = do act <- elabScript fc nest env
                                 !(evalClosure defs act) exp
```

Comment:
This `exp` is suspicious: it's the type of the overall elab script, not of `act`.
```suggestion
                                !(evalClosure defs act) Nothing
```